academic-services/graph/resources-faq.md

```diff
@@ -59,4 +59,10 @@ It is relatively easy and inexpensive to compute citation counts and h-index usi
 
 ## What is FamilyId in Papers.txt
 
-Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value  being the PaperId of the main paper.
+Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value being the PaperId of the main paper.
+
+## PaperCount vs. PaperFamilyCount
+
+PaperCount is the number of papers associated with the entity.
+
+PaperFamilyCount is similar as PaperCount where multiple papers with same FamilyId are only counted as one.
```

Comment:
>PaperFamilyCount is similar as PaperCount where multiple papers with same FamilyId are only counted as one [](start = 0, length = 106)

If we change the above definition, we could simply make this "PaperFamilyCount is the number of primary family papers associated with the entity."